components/contact/inquiry.coffee

```diff
@@ -22,7 +22,7 @@ module.exports = class InquiryView extends ContactView
     formTemplate _.extend locals,
       artwork: @artwork
       user: @user
-      contactGallery: false
+      contactGallery: @model.contact_gallery
```

Comment:
what's `@model` here, it's not the `@artwork`? Should it be `@model.get('contact_gallery')`?